### ISSUE TITLE
EACCES node error

### ISSUE THREAD

**Issue by finnhodgkin** [dwyl/learn-amazon-web-services#20]
The current instructions for creating a simple node server throw an error: 

![error](https://user-images.githubusercontent.com/22300773/28016579-e3833684-656c-11e7-9643-8427d662a616.gif)

**Comment by finnhodgkin** [comment]
I installed node via NVM so it doesn't have access rights to port 80. Experimenting with proxy-ing the port now.

**Comment by ZooeyMiller** [comment]
If you don't install node as root (which as far as I understand is preferable) running things on port `80` doesn't work. 
As a result you have to redirect any request from port 80 to a different port in your remote ubuntu instance. This can be done with this command:
```
sudo iptables -t nat -I PREROUTING -p tcp --dport 80 -j REDIRECT --to-port 3000
```

and change the port on the server to `3000` as well.

**Comment by finnhodgkin** [comment]
fixed in #22
